sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Verifier.cs

```diff
@@ -120,27 +120,33 @@ internal static class Verifier
             ParseOptions options = null, params MetadataReference[] additionalReferences)
         {
             VerifyAnalyzer(new[] { new DocumentInfo($"file1{CSharpFileExtension}", snippet) },
-                CSharpFileExtension, diagnosticAnalyzer, options, null, additionalReferences);
+                CSharpFileExtension, new[] { diagnosticAnalyzer }, options, null, additionalReferences);
         }
 
         public static void VerifyVisualBasicAnalyzer(string snippet, SonarDiagnosticAnalyzer diagnosticAnalyzer,
             ParseOptions options = null, params MetadataReference[] additionalReferences)
         {
             VerifyAnalyzer(new[] { new DocumentInfo($"file1{VisualBasicFileExtension}", snippet) },
-                VisualBasicFileExtension, diagnosticAnalyzer, options, null, additionalReferences);
+                VisualBasicFileExtension, new[] { diagnosticAnalyzer }, options, null, additionalReferences);
         }
 
         public static void VerifyAnalyzer(string path, SonarDiagnosticAnalyzer diagnosticAnalyzer,
             ParseOptions options = null, params MetadataReference[] additionalReferences)
         {
-            VerifyAnalyzer(new[] { path }, diagnosticAnalyzer, null, options, additionalReferences);
+            VerifyAnalyzer(new[] { path }, new[] { diagnosticAnalyzer }, null, options, additionalReferences);
+        }
+
+        public static void VerifyAnalyzer(string path, IEnumerable<SonarDiagnosticAnalyzer> diagnosticAnalyzers,

```

Comment:
Adding overload that allows you to run two analyzers in one test. This was needed for the InvalidCast rule which was split into two.